hack/test/e2e-qemu.sh

```diff
@@ -275,6 +269,7 @@ function create_cluster {
     --cidr=172.20.1.0/24 \
     --install-image="${INSTALLER_IMAGE}" \
     --with-init-node=false \
+    --with-json-logs=true \
```

Comment:
we can do this, but I'm not super excited about it, as this changes the defaults for all runs, and makes our CI run as if logs are always enabled, so we don't test the case when the logs are not enabled. 

so I would leave at least a single job with disabled logs

Reply:
I do not know, but a job that never reboots should be the perfect candidate to do this, both testing without the option and not losing any warnings. Do you know such jobs?

Reply:
Any integration-misc might be fine, like a disk image one.